web/src/containers/SummaryBudgetByActivity.js

```diff
@@ -0,0 +1,184 @@
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import Instruction from '../components/Instruction';
+import SummaryActivityBreakdownTable from './SummaryActivityBreakdown';
+import Dollars from '../components/Dollars';
+
+const categoryLookup = {
+  statePersonnel: { title: 'State Staff Total', sortOrder: 0 },
+  expenses: { title: 'Other State Expenses Total', sortOrder: 1 },
+  contractors: { title: 'Private Contractor Total', sortOrder: 2 },
+  combined: { title: 'Total', sortOrder: 3 }
+};
+
+const categorySort = (key1, key2) =>
+  categoryLookup[key1].sortOrder - categoryLookup[key2].sortOrder;
+
+const DataRow = ({ data, title, groupTitle }) => (
+  <tr
+    className={
+      title === categoryLookup.combined.title
+        ? 'budget-table--subtotal budget-table--row__highlight'
+        : ''
+    }
+  >
+    <th scope="row" className="title indent-title">
+      {title === categoryLookup.combined.title
+        ? `${groupTitle} ${title}`
+        : title}
+    </th>
+    <td className="budget-table--number">
+      <Dollars>{data.medicaid}</Dollars>
+    </td>
+  </tr>
+);
+
+DataRow.propTypes = {
+  data: PropTypes.object.isRequired,
+  title: PropTypes.string.isRequired,
+  groupTitle: PropTypes.string.isRequired
+};
+
+const DataRowGroup = ({ data, year, groupTitle }) => (
+  <tbody>
+    {Object.keys(data)
+      .sort(categorySort)
+      .map(key => (
```

Comment:
oh, duh.  It took me a while to flip this around in my head but then when I did the light bulb came on!!!

Reply:
It seems like one of my comments disappeared when I clicked the submit button, because there's supposed to be another one before this (or else I just can't find it again) that would have explained it better. Sorry this ended up being more confusing than I meant it!

Reply:
😂